packages/fxa-auth-server/lib/senders/emails/templates/subscriptionSubsequentInvoice/index.stories.ts

```diff
@@ -53,7 +53,7 @@ export const SubscriptionSubsequentInvoicePayPalNoProrated = createStory(
 
 export const SubscriptionSubsequentInvoiceStripeProrated = createStory(
   {
-    cardType: 'MasterCard',
+    cardType: 'Mastercard',
```

Comment:
Not a regression, but because my eyes see it, why is it `cardType: 'mastercard'` (lowercase "m") above in: 

https://github.com/mozilla/fxa/blob/17a0cd6555fed58f4a0662a81e5e1564fbe29d88/packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoice/index.stories.ts#L128-L129

&hellip; but here its `cardType: 'Mastercard'` (uppercase "M")?
But they're also both *.stories.ts files, so 🤷.